//package candy.protocol;
//Gateway.php 
message MseGatewayFunction {
    optional int32 action_idx =1;
    optional string ServerName =2;
    optional string	ActionName =3;
    optional string JsonResult =4;
    optional int32 err=5;
    optional string dbm=6;
    optional int32 TotalPage = 7[default=1];
    optional int32 CurrPage  = 8[default=1];
}
